onnxruntime/python/_pybind_state.py.in

```diff
@@ -25,7 +24,7 @@ if platform.system() == "Windows":
         if not os.path.isfile(os.path.join(system_root, "System32", "vcruntime140_1.dll")):
             warnings.warn("Please install the 2019 Visual C++ runtime and then try again. "
```

Comment:
Starting from the next release, we will require VS 2022 runtime instead.

Reply:
There are many files mentioned VS 2019, we can update them together in a batch.